src/toil/batchSystems/singleMachine.py

```diff
@@ -120,6 +120,33 @@ def __init__(self, config, maxCores, maxMemory, maxDisk):
             self.workerThreads.append(worker)
             worker.start()
 
+    def _get_debug_cmd(self, jobCommand):
```

Comment:
Thanks Hannes, fixed. Apologies, it's quite hard to get used to camelCase when you normally use the pep8 styles.